src/metrics/index.js

```diff
@@ -0,0 +1,41 @@
+const fs = require('fs');
+const HttpClient = require("@actions/http-client").HttpClient
+const AmbassadorClient = require('../ambassador/api-client');
+const core = require('@actions/core');
+const configure = require('../configure');
+
+const httpClient = new HttpClient();
+const ambassadorClient = new AmbassadorClient(process.env.TELEPRESENCE_API_KEY)
+
+class MetritonClient {
+    static async sendMetricsReport(action){
+        try {
+            const reportDestination = 'https://metriton.datawire.io/beta/scout'
+            const applicationName = 'telepresence-github-action-integration'
+            const installId = fs.readFileSync(`${configure.getTelepresenceConfigPath()}/id`).toString()
```

Comment:
This is already handled on the try catch block, SendReport will fail. Unless you have a specific suggestion I am resolving this so we can merge since this is not a blocker. We can handle this on another PR.